### ISSUE TITLE
Generalize manifest generator implementation to support 1) BOTH CVM execution and appraisal, 2) Multiple CVM executions/appraisals

### ISSUE THREAD

**Issue by ampetz** [ku-sldg/copland-avm#37]
At the moment there are two implementations of the manifest generator:  one for CVM execution (attestation) and one for appraisal.  They are quite similar, and have similar properties proven about them.  These should likely be merged into a single generator that does both simultaneously.  The main challenge will be that attestation version takes a term as input, whereas the appraisal version takes an Evidence type.  

Tasks:

- [ ] Merge outputs of attestation and appraisal manifest generation
- [ ] Merge outputs of multiple attestation/appraisal generation

Blocked by #42
